cogeo_mosaic/mosaic.py

```diff
@@ -275,7 +275,13 @@ def from_urls(
             >>> MosaicJSON.from_urls(["1.tif", "2.tif"])
 
         """
-        features = get_footprints(urls, max_threads=max_threads, quiet=quiet)
+        tms = kwargs.pop('tilematrixset', WEB_MERCATOR_TMS)
```

Comment:
can we add the `tms` option in the function definition

same as in rio-cogeo https://github.com/cogeotiff/rio-cogeo/blob/main/rio_cogeo/cogeo.py#L86
https://github.com/cogeotiff/rio-cogeo/blob/main/rio_cogeo/cogeo.py#L194

Reply:
went with `tilematrixset` as the parameter name, otherwise other usages like in test_mosaic_create_tms are not valid and would require updates, so hopefully less of a breaking change this way. That said the inconsistency between some functions expecting tms and other expecting tilematrixset is problematic

Reply:
@vincentsarago do you think it's worth renaming the parameter to `tms` here and elsewhere, the inconsistency is something I'd like to correct, but have no strong leaning which name to pick other than a preference for being explicit